KarhooUISDK/FeatureFlags/FeatureFlagsUpdater.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  FeatureFlagsUpdater.swift
+//  KarhooUISDK
+//
+//  Created by Bartlomiej Sopala on 15/04/2023.
+//  Copyright © 2023 Flit Technologies Ltd. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class FeatureFlagsUpdater {
+    
+    private let currentSdkVersion: String
+    private let featureFlagsStore: FeatureFlagsStore
+    
+    init(
+        currentSdkVersion: String,
+        featureFlagsStore: FeatureFlagsStore = KarhooFeatureFlagsStore()
+    ) {
+        self.currentSdkVersion = currentSdkVersion
+        self.featureFlagsStore = featureFlagsStore
+    }
+    
+    func update() {
```

Comment:
What are your thoughts on ensuring this whole update flow is performed on utility QoC?